components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/util/Utils.java

```diff
@@ -0,0 +1,269 @@
+import static org.wso2.carbon.identity.organization.management.service.constant.OrganizationManagementConstants.ORGANIZATION_PATH;
+import static org.wso2.carbon.identity.organization.management.service.constant.OrganizationManagementConstants.PATH_SEPARATOR;
+import static org.wso2.carbon.identity.organization.management.service.constant.OrganizationManagementConstants.SERVER_API_PATH_COMPONENT;
+import static org.wso2.carbon.identity.organization.management.service.constant.OrganizationManagementConstants.V1_API_PATH_COMPONENT;
+import static org.wso2.carbon.identity.organization.management.service.constant.SQLConstants.MICROSOFT;
+import static org.wso2.carbon.identity.organization.management.service.constant.SQLConstants.ORACLE;
+
+/**
+ * This class provides utility functions for the Organization Management.
+ */
+public class Utils {
+
+    private static DataSource dataSource;
+
+    /**
+     * Throw an OrganizationManagementClientException upon client side error in organization management.
+     *
+     * @param error The error enum.
+     * @param data  The error message data.
+     * @return OrganizationManagementClientException
+     */
+    public static OrganizationManagementClientException handleClientException(
+            OrganizationManagementConstants.ErrorMessages error, String... data) {
+
+        String description = error.getDescription();
+        if (ArrayUtils.isNotEmpty(data)) {
+            description = String.format(description, data);
+        }
+        return new OrganizationManagementClientException(error.getMessage(), description, error.getCode());
+    }
+
+    /**
+     * Throw an OrganizationManagementServerException upon server side error in organization management.
+     *
+     * @param error The error enum.
+     * @param e     The error.
+     * @param data  The error message data.
+     * @return OrganizationManagementServerException
+     */
+    public static OrganizationManagementServerException handleServerException(
+            OrganizationManagementConstants.ErrorMessages error, Throwable e, String... data) {
+
+        String description = error.getDescription();
+        if (ArrayUtils.isNotEmpty(data)) {
+            description = String.format(description, data);
+        }
+        return new OrganizationManagementServerException(error.getMessage(), description, error.getCode(), e);
+    }
+
+    /**
+     * Get a new Jdbc Template.
+     *
+     * @return a new Jdbc Template.
+     */
+    public static NamedJdbcTemplate getNewTemplate() throws OrganizationManagementServerException {
+
+        try {
+            if (dataSource == null) {
+                dataSource = DatabaseUtil.getRealmDataSource(CarbonContext.getThreadLocalCarbonContext().getUserRealm()
+                        .getRealmConfiguration());
+            }
+        return new NamedJdbcTemplate(dataSource);
```

Comment:
Shall we fix the indentation here

Reply:
Resolved by 1967acd79b1051fb7f1d64158b5d8671b7047b9d